nix/ucm.nix

```diff
@@ -100,10 +100,10 @@ in
     '';
 
     postFixup = ''
-      installShellCompletion --cmd ucm \
-        --bash <(${ucm} --bash-completion-script ${ucm}) \
-        --fish <(${ucm} --fish-completion-script ${ucm}) \
-        --zsh <(${ucm} --zsh-completion-script ${ucm})
+      # installShellCompletion --cmd ucm \
+      #   --bash <(${ucm} --bash-completion-script ${ucm}) \
```

Comment:
Afterwards, I tried
```
ucm --bash-completion-script
```

but that doesn't return anything which could be why these are empty?